src/utils/AllStopUtils.js

```diff
@@ -74,7 +74,8 @@ function fetchPrecisionMaps() {
  */
 async function getPrecisionMap(degreesPrecision: ?number = Constants.DEG_EQ_PRECISION) {
   if (degreesPrecision < Constants.DEG_MIN_PRECISION
```

Comment:
newline after the initial `if (` is cleaner imo but your call

Reply:
I agree with @kq-li here!